ui_framework/components/tabs/tabs.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTab } from './tab';
+
+export const KuiTabs = ({
+  children,
+  selectedTabIndex,
+  onSelectedTabChanged,
+  className,
+  ...rest
+}) => {
+  const classes = classNames('kuiTabs', className);
+  const tabs = children.map((child,index)=>(
```

Comment:
The linter should have complained about the missing space after the comma. :thinking: